eglib/src/glib.h

```diff
@@ -486,6 +486,7 @@ struct _GByteArray {
 GByteArray *g_byte_array_new    (void);
 GByteArray* g_byte_array_append (GByteArray *array, const guint8 *data, guint len);
 guint8*  g_byte_array_free      (GByteArray *array, gboolean free_segment);
+void     g_byte_array_set_size  (GByteArray *array, gint length);
```

Comment:
It does exist in Glib, see https://developer.gnome.org/glib/2.28/glib-Byte-Arrays.html#g-byte-array-set-size.